public/app/features/trails/AutomaticMetricQueries/units.ts

```diff
@@ -14,6 +14,19 @@ export function getUnitFromMetric(metric: string) {
   }
 }
 
+// Get unit from recording rule (e.g. "workload:memory_working_set_bytes:sum" -> "bytes")
+export function getUnitFromRecordingRule(recordingRule: string): string | null {
+  const units = ['bytes', 'seconds'];
+  const lowerCaseRule = recordingRule.toLowerCase();
+
+  for (const unit of units) {
+    if (lowerCaseRule.includes(unit)) {
+      return unit;
+    }
+  }
+  return null;
+}
```

Comment:
I think we can use units from a general place. I see we have `UNIT_LIST` so let's use it. 
Also, I'd use `.find()` instead of for loop. https://www.w3schools.com/jsref/jsref_find.asp
It returns `undefined` though. But I think we can just change the function signature and it will be fine. 
WDT?

Reply:
is the benefit of using `.find()` instead of for loop that we don't have to iterate over the entire recording rule to look for a unit name? i.e., once we find "bytes" or "seconds" we can stop looking over the recording rule?